### PR TITLE
Updated README.md with minor edit on the Project Board

### PR DESCRIPTION
**Pull request by vidya-mohan**
**Issue #:**
None

**Description of changes:**
Updated README.md with minor edit on rendering of Project Board

**Testing**

1. Have you successfully run `npm run build:release` locally?
2. How did you test these changes?
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Amazon Chime SDK for JavaScript
 [Amazon Chime SDK React Components](https://github.com/aws/amazon-chime-sdk-component-library-react)
-[Project Board](https://aws.github.io/amazon-chime-sdk-js/modules/projectboard.html)
+
+[Amazon Chime SDK Project Board](https://aws.github.io/amazon-chime-sdk-js/modules/projectboard.html)
```
Comment (devalevenkatesh) [inline README.md:4]: How about moving this above the React components link since this is the project board?

**Review by devalevenkatesh — COMMENTED** [review]
I have few comments in the earlier merged PR, can we update those as well?
[1](https://github.com/aws/amazon-chime-sdk-js/pull/1431#discussion_r670603767), [2](https://github.com/aws/amazon-chime-sdk-js/pull/1431#discussion_r670610127), [3](https://github.com/aws/amazon-chime-sdk-js/pull/1431#discussion_r670613833).

File: README.md (hunk 1 above)
Comment (vidya-mohan) [reply README.md:4]: sounds good

**Comment by michhyun1** [comment]
LGTM

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**